src/js/components/Widgets/ScoreDisplay/PositionItemScorePopover.jsx

```diff
@@ -245,9 +239,8 @@ class PositionItemScorePopover extends Component {
             <StatementText>
               <Suspense fallback={<></>}>
                 <ReadMore
-                  textToDisplay={`"${statementText}"`}
+                  textToDisplay={`"9 ${statementText}"`}
```

Comment:
Hello @stilwellmg, please make sure to remove the '9 ' which was only included to help find the code behind what we saw on the screen.